src/lib/setup/fsUtil.ts

```diff
@@ -0,0 +1,33 @@
+import fs from "fs-extra";
+import { removeDir } from "../ioUtil";
+
+/**
+ * Change directory to an absolute path
+ *
+ * @param absPath Absolute Path
+ */
+export const moveToAbsPath = (absPath: string) => {
```

Comment:
Just commenting to say `fsUtil` could probably live in our general lib as could be shared by other commands too.

Reply:
we can move it later if needed.